CoreService/Trustees.psm1

```diff
@@ -90,6 +90,53 @@ function Get-User
 	}
 }
 
+function Get-Group
+{
+    <#
+    .Synopsis
+    Gets information about the a specific Tridion group.
+
+    .Description
+    Gets a group object containing information about the specified group within Tridion.
+
+    .Inputs
+    None.
+
+    .Outputs
+    Returns an object of type [Tridion.ContentManager.CoreService.Client.TrusteeData].
+
+    .Link
+    Get the latest version of this script from the following URL:
+    https://github.com/pkjaer/tridion-powershell-modules
+
+    .Example
+    Get-TridionGroup "WebMaster"
+    Returns information about the group 'WebMaster'.
+	
+	.Example
+    Get-TridionUser "tcm:2-12-65552"
+    Returns information about group with the id 'tcm:2-12-65552'.
+    
+    #>
+    [CmdletBinding()]
+    Param
+    (
+        [Parameter(ValueFromPipelineByPropertyName=$true)]
+		[ValidateNotNullOrEmpty()]
+        [string]$Group
+    )
+
+	Write-Verbose "Loading Tridion group...";
+	$GroupObj = Get-Groups | ?{($group -eq ($_.Id)) -or ($group -eq $_.Title)} | Select -First 1
```

Comment:
That actually makes sense, since that's how Tridion works on the internals. the `$client.IsExistingObject` and `$client.Read` functions are working with both tcm ids and webdav urls as a correct "id".
Actually that means that `Get-TridionItem` probably also supports both types.